### DISCUSSION TITLE
Parent child relationships example

### DISCUSSION THREAD

**Discussion by manosbatsis** [refinedev/refine#5267] [Q&A] [answered]
Are there any howto approaches and/or examples out there on  properly implementing parent-child resource relationships in Refine UIs? Specifically interested in:

- Routes for _parents/:parentId/children_ show|list|crud
- Parent _show_ pages that embed a children _list_ view with proper create(add) functionality and redirects back to the parent resource 
- _Create_ view form( field)s aware of the parent context or just picking up the parent id from the URL

**Comment by alicanerdurmaz** [comment, marked as answer]
Hello @manosbatsis,

You can check out these examples
- https://refine.dev/docs/advanced-tutorials/multi-tenancy/strapi-v4/#what-is-multitenancy
- https://refine.dev/docs/advanced-tutorials/multi-tenancy/appwrite/#what-is-multitenancy
- https://refine.dev/docs/advanced-tutorials/multi-level-menu/

**Reply by manosbatsis** [reply]
My knee jerk reaction was to dismiss these as irrelevant, but I see now how these examples are useful. Thanks a lot  @alicanerdurmaz, will try this approach.
